build.py

```diff
@@ -13,44 +13,58 @@
 import os
 import urllib.request
 
+# version of lucene we build against
+LUCENE_VERSION = '6.4.2'
+
 deps = [
   #('org.codehaus.jackson', 'jackson-core-asl', '1.9.13'),
   #('org.codehaus.jackson', 'jackson-mapper-asl', '1.9.13'),
   ('com.fasterxml.jackson.core', 'jackson-core', '2.8.2'),
   #('com.fasterxml.jackson.core', 'jackson-mapper', '2.8.2'),
   ('commons-codec', 'commons-codec', '1.10'),
-  ('net.minidev', 'json-smart', '1.2')
+  ('net.minidev', 'json-smart', '1.2'),
+  ('org.apache.lucene', 'lucene-core', LUCENE_VERSION),
+  ('org.apache.lucene', 'lucene-analyzers-common', LUCENE_VERSION),
+  ('org.apache.lucene', 'lucene-analyzers-icu', LUCENE_VERSION),
+    ('com.ibm.icu', 'icu4j', '56.1'),
+  ('org.apache.lucene', 'lucene-facet', LUCENE_VERSION),
+  ('org.apache.lucene', 'lucene-codecs', LUCENE_VERSION),
+  ('org.apache.lucene', 'lucene-grouping', LUCENE_VERSION),
+  ('org.apache.lucene', 'lucene-highlighter', LUCENE_VERSION),
+  ('org.apache.lucene', 'lucene-join', LUCENE_VERSION),
+  ('org.apache.lucene', 'lucene-misc', LUCENE_VERSION),
+  ('org.apache.lucene', 'lucene-queries', LUCENE_VERSION),
+  ('org.apache.lucene', 'lucene-queryparser', LUCENE_VERSION),
+  ('org.apache.lucene', 'lucene-suggest', LUCENE_VERSION),
+  ('org.apache.lucene', 'lucene-expressions', LUCENE_VERSION),
+    ('org.antlr', 'antlr4-runtime', '4.5.1-1'),
+    ('org.ow2.asm', 'asm', '5.1'),
+    ('org.ow2.asm', 'asm-commons', '5.1'),
+  ('org.apache.lucene', 'lucene-replicator', LUCENE_VERSION),
+    ('org.apache.httpcomponents', 'httpclient', '4.4.1'),
```

Comment:
Whoa, cool -- you did transitive closure, manually, here, so we do not even look at `ant` build files anymore from Lucene?